openedx/core/djangoapps/programs/tests/test_views.py

```diff
@@ -245,3 +255,44 @@ def test_get_course_programs_with_invalid_response(self, log_exception):
                 program
             )
             self.assertEqual(programs, {})
+
+    @ddt.data(0, 100, 200)
```

Comment:
Testing with both 100 and 200 is redundant - the test is doing the same thing since the duration above 0 is not considered.  I would actually suggest testing with 0 and 1 (or 2, if 1 is flaky).